### PR TITLE
Handle google regions for image queries (CSE-65)

### PR DESCRIPTION
**Pull request by chajain**
when querying the public-cloud-info-service the region should be ignored when the CSP is Google

### CODE REVIEW

File: spec/features/app_spec.rb [hunk 1]
```diff
@@ -613,3 +613,16 @@
     end
   end
 end
+
+# we test the region images queries for google
+describe 'google region images query' do
+  describe 'google images' do
+    before do
+      @path = '/v1/google/europe-north1/images/active.xml'
```
Comment (guangyee) [inline spec/features/app_spec.rb:621]: What happens if there are two regions with different active.xml? Or we don't have such scenario?

Comment (jeremy-moffitt) [reply spec/features/app_spec.rb:621]: my understanding is that the Google images are global, so there is no region metadata for them, which causes the list to come back empty if a region is specified. See https://github.com/SUSE-Enceladus/public-cloud-info-service/issues/8  , so in this case if there are multiple regions, their active images should be the same.

**Review by guangyee — APPROVED** [review]
Sounds good.

File: app.rb [hunk 2]
```diff
@@ -261,6 +261,9 @@ def form_xml_for_settings(tag, values)
         .in_state(params[:image_state])
         .in_env(environment)
         .set_region(params[:region])
+    elsif params[:provider] == 'google'
```
Comment (bear454) [inline app.rb:264]: I don't like this being keyed on `'google'`; this should be a more generic behavior for any images that do not have a defined region or environment.

**Review by bear454 — CHANGES_REQUESTED** [review]
Please make this more general for the case of non-regioned images instead of specific to provider==google.

File: app.rb (hunk 2 above)
Comment (rjschwei) [reply app.rb:264]: Yep, same applies to Oracle and this is related to #9 when we have global images and no update infrastructure.

File: spec/features/app_spec.rb (hunk 1 above)
Comment (rjschwei) [reply spec/features/app_spec.rb:621]: There is always only 1 active image for a given image type. For CSPs where images are global (Google and Oracle) the global image names that match the filters should be returned when a specific region is queried.

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: app.rb (hunk 2 above)
Comment (chajain) [reply app.rb]: I have updated the code so that no longer google is no longer hard coded

**Review by rjschwei — APPROVED** [review]

**Review by bear454 — APPROVED** [review]

**Review by jeremy-moffitt — APPROVED** [review]
LGTM, thanks!

**Pull request merged**